process/operations/operationsProcessor.go

```diff
@@ -0,0 +1,70 @@
+package operations
+
+import (
+	indexer "github.com/ElrondNetwork/elastic-indexer-go"
+	"github.com/ElrondNetwork/elastic-indexer-go/data"
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	"github.com/ElrondNetwork/elrond-go-core/data/transaction"
+)
+
+type operationsProcessor struct {
+	importDBMode     bool
+	shardCoordinator indexer.ShardCoordinator
+}
+
+// NewOperationsProcessor will create a new instance of operationsProcessor
+func NewOperationsProcessor(importDBMode bool, shardCoordinator indexer.ShardCoordinator) (*operationsProcessor, error) {
+	if check.IfNil(shardCoordinator) {
+		return nil, indexer.ErrNilShardCoordinator
+	}
+
+	return &operationsProcessor{
+		shardCoordinator: shardCoordinator,
```

Comment:
maybe keep the same order as the one declared

Reply:
done.